### ISSUE TITLE
Not all items appear in the Create tab

### ISSUE THREAD

**Issue by Maciek103** [mrh0/createaddition#772]
**Describe the bug**
Not all items appear in the Create tab while you are on creative

**To Reproduce**
I'm using Create, extended bogeys, SnR and Create Stuff & Additions, and sometimes not all item shows in tab

**Additional context**
- Mod version: 1.2.3
- Forge version: 47.2.20
- Create version: 0.5.1.f
![i321mage](https://github.com/mrh0/createaddition/assets/45295911/851ed11d-b52c-4a2b-ab20-553eaff2435c)
![2024-04-08 22_26_11-Minecraft Forge_ 1 20 1 - Singleplayer](https://github.com/mrh0/createaddition/assets/45295911/2f89756a-5080-4588-acc9-405495cebf06)
![2024-04-08 22_26_03-Minecraft Forge_ 1 20 1 - Singleplayer](https://github.com/mrh0/createaddition/assets/45295911/1dbd1349-68a0-43d8-ad64-f28e01a5e465)
![312](https://github.com/mrh0/createaddition/assets/45295911/4936543d-b507-4eaf-8d75-e3fbea118a62)

**Comment by Maciek103** [comment]
Without create additions everything show every time

**Comment by mrh0** [comment]
I'm not sure what I'm supposed to see in these images, whats missing? could you elaborate?

**Comment by Maciek103** [comment]
Yes, that's my bad I attached wrong screenshot. But now I don't know why this bug doesn't always appear, only once every 10 times I start the game, as soon as it appears I will post a screenshot.

**Comment by Maciek103** [comment]
But wait, I attached right screenshot just look, on the first there is no shaft, and something else but I don't remember what was that

**Comment by Maciek103** [comment]
https://github.com/Creators-of-Create/Create/issues/6222

There is more about that issue and I think the solution but I don't understand that
